qa/L0_response_cache/test.sh

```diff
@@ -272,15 +328,71 @@ run_server
 check_server_expected_failure "WRONGPASS"
 unset TRITONCACHE_REDIS_USERNAME
 unset TRITONCACHE_REDIS_PASSWORD
-
-# Clean up redis server before exiting test
+# Clean up redis server
 unset_redis_auth
 stop_redis
 
+
+#Test Ensemble Model With Cache Enabled and Decoupled Mode in Ensemble Config
+CACHE_FUNCTION="EnsembleCacheTest.setup_cache_ensemble_model"
+python ${ENSEMBLE_CACHE_TEST_PY} ${CACHE_FUNCTION} >> ${CLIENT_LOG} 2>&1
+DECOUPLED_FUNCTION="EnsembleCacheTest.setup_decoupled_ensemble_model"
+python ${ENSEMBLE_CACHE_TEST_PY} ${DECOUPLED_FUNCTION} >> ${CLIENT_LOG} 2>&1
+check_server_failure_decoupled_model ${ENSEMBLE_MODEL_DIR} ${ENSEMBLE_MODEL}
+RESET_CONFIG_FUNCTION="EnsembleCacheTest.reset_config"
+python ${ENSEMBLE_CACHE_TEST_PY} ${RESET_CONFIG_FUNCTION} >> ${CLIENT_LOG} 2>&1
+
+
+# Test Ensemble Model With Cache Enabled and Decoupled Mode in Composing Model
```

Comment:
ditto here and similar following comments to the tests: I'd prefer capitalization of only the first letter

Reply:
and a space after `#`